tensorboard/webapp/widgets/markdown_renderer/markdown_renderer_component.css

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+markdown-renderer > p:first-child {
```

Comment:
Actually drop `>`? There won't be a paragraph immediately. I think it should be `markdown-renderer .content > p:first-child` after putting a class on the `div`.

Reply:
what about for last-child?  Bit unfamiliar with this css syntax.  Also, would you like me to place the content class on the div in  `markdown_renderer_component.ng.html`?

Reply:
This comment does apply to `:last-child` too. 

If you are referring to `>` syntax, it means we are `<p>` that is an immediate child of `<markdown-renderer>` (e.g., if there are nested paragraphs, we won't apply the style here). However, if you look at the DOM structure, there is a `div` in between possible `p` and `markdown-renderer`. 

Yes, please do add a class on the `div` on the markdown_renderer_component.ng.html file.